reference_docs/YAML_models/standoc_document_attributes.yaml

```diff
@@ -0,0 +1,724 @@
+    value_default: Latn
+
+  boilerplate-authority:
+    tags: ["Languages and localization"]
+    title: "Boilerplate authority"
+    description: "File containing predefined text of document, in Metanorma XML. The document predefined text needs to follow the structure described in link:/builder/topics/metadata-and-boilerplate#boilerplate[Predefined text]; compare examples of Metanorma predefined text files such as https://github.com/metanorma/metanorma-itu/blob/main/lib/asciidoctor/itu/boilerplate.xml[that in ITU]".
+    added_in_version: v1.3.15
+
+  #--- Document info ---#
+  publisher:
+    tags: ["Document info"]
+    title: "Publisher"
+    description: "Standards agency publishing the standard; can be multiple (semicolon-delimited: processed via CSV, recognising quote marks)".
+    value_spec: comma-list
+    added_in_version: v1.5.1
+
+
+  copyright-holder:
+    tags: ["Document info"]
+    title: "Copyright holder"
+    description: "Copyright holder, if distinct from the publisher; can be multiple (semicolon-delimited: processed via CSV, recognising quote marks)".
+    value_spec: comma-list
+    added_in_version: v1.5.1
+
+  docnumber:
+    tags: ["Document info"]
+    title: "Document number"
+    description: "Numeric component of the document identifier. The full identifier is formed by prefixing and suffixing this element with other strings derived from metadata".
+
+  edition:
+    tags: ["Document info"]
+    title: "Edition"
+    description: "Document edition".
+
+  revdate:
+    tags: ["Document info"]
+    title: "Date revision"
+    description: "Date the document was last updated".
+
+  library-ics:
+    tags: ["Document info"]
+    title: "Library ICS"
+    description: "ICS (International Categorization for Standards) number for the standard. There may be more than one ICS for a document; if so, they should be comma-delimited. (The ICS identifier is added to the document metadata, but may not be visible in the resulting document, depending on Metanorma flavor.)".
+
+  isbn:
+    tags: ["Document info"]
+    title: "ISBN-13 number"
+    description: "ISBN-13 number of the document".
+    added_in_version: v1.6.2
+
+  isbn10:
+    tags: ["Document info"]
+    title: "ISBN-10 number"
+    description: "ISBN-10 number of the document".
+    added_in_version: v1.6.2
+
+  title:
+    tags: ["Document info"]
+    title: "Title of the document"
+    description: "Title of the document. If not supplied, the built-in AsciiDoc title (first line of document header) is used instead.".
+
+  title-XX:
+    tags: ["Document info"]
+    title: "Title of document in certain language"
+    description: "Title of the document in the language `XX` (where “XX” is a ISO 639-1 code; for example, `:title-en:`, `:title-fr`:).".
+    declaration: ":title-en:, :title-fr:"
+
+  doctype:
+    tags: ["Document info"]
+    title: "Document type"
+    description: 'The document type; e.g. "standard", "guide", "report"'.
```

Comment:
> Please add the document types that are defined in Standoc in value_spec

Maybe I'm mistaken, but I think it would be impractical to do this in this case because I understand that Standoc attributes are supported by several flavors of Metanorma. So, I would have to specify the values in each flavor. Right @opoudjis?

Reply:
Correct. The document types are not defined at standoc level, but in concrete flavours.